Cargo.toml

```diff
@@ -14,3 +14,5 @@ members = [
 
 [profile.release]
 lto = true
+opt-level = 's'
```

Comment:
If we're doing precompiles, size should not matter as much since my understanding is these system contracts will be stored locally. Gas costs are fixed for execution, might as well optimize for performance.